atlas/storage/internal/model/models.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+import "time"
+
+type Note struct {
+	Service string `gorm:"primaryKey"`
+	Mode    bool
+}
+
+type Service struct {
+	ServiceName          string
+	ServiceDesc          string
+	ServiceUptime        time.Time
+	ServiceCountRequests uint
+}
+
+type Msg struct {
+	Id      string
+	Command string
+}
+
+type StorageResponse struct {
+	Id    string
+	Value interface{}
+}
```

Comment:
If you are using the same structs in different services, it's better not to re-implement these structs in each service, but import package `models` (with these shared structs) from other service

Reply:
i understand this issue, but what if i will divide this repo for some separate repos ?